2018-06-23--03:43:59
Traceback (most recent call last):
  File "D:/MyPythonProject/LearnPython/day01-语法结构/11.异常处理.py", line 13, in <module>
    print(name[2])
IndexError: list index out of range
2018-06-23--03:44:07
Traceback (most recent call last):
  File "D:/MyPythonProject/LearnPython/day01-语法结构/11.异常处理.py", line 13, in <module>
    print(name[3])
IndexError: list index out of range
2018-06-23--03:44:20
Traceback (most recent call last):
  File "D:/MyPythonProject/LearnPython/day01-语法结构/11.异常处理.py", line 13, in <module>
    print(name[4])
IndexError: list index out of range
2018-06-23--03:44:32
Traceback (most recent call last):
  File "D:/MyPythonProject/LearnPython/day01-语法结构/11.异常处理.py", line 13, in <module>
    1/0
ZeroDivisionError: division by zero
2018-07-10--19:48:16
Traceback (most recent call last):
  File "D:/MyPythonProject/LearnPython/day01-语法结构/11.异常处理.py", line 13, in <module>
    1/0
ZeroDivisionError: division by zero
